lib/releaselib.sh

```diff
@@ -57,11 +57,10 @@ release::update_job_cache () {
     # the cache should be complete after that
     [[ -n ${JOB[$run]} ]] && break
 
-    buildlog=$($GSUTIL cat $logroot/$job/$run/build-log.txt 2>/dev/null)
-    if [[ "$buildlog" =~ Uploading\ build\ result:\ SUCCESS ]] &&
-       [[ "$buildlog" =~ \
-          (build_version=|Found Kubernetes version: )(${VER_REGEX[release]}\.${VER_REGEX[build]}) ]]; then
-      JOB[$run]=${BASH_REMATCH[2]}
+    buildstate=$($GSUTIL cat $logroot/$job/$run/*.json 2>/dev/null)
+    if [[ "$buildstate" =~ \"result\":\ \"SUCCESS\", ]] &&
```

Comment:
I'm probably splitting hairs with my desire to keep external calls to a minimum:
```
$ time echo "$foo" |jq -r '.version,.result'
null
SUCCESS
v1.3.0-alpha.5.216+66149c50fd0055
null

real	0m0.003s
user	0m0.000s
sys	0m0.007s
```

Change coming...